src/mame/swtpc/swtpc09.cpp

```diff
@@ -68,23 +68,90 @@
  F100 - ????  CDS disk 0 controller
  F300 - ????  CDS disk 1 controller
 
-The supplied flex and uniflex floppies all worked when this driver was
-created (2014-03-25), but have been broken since sometime in 2015.
-
-The message is the same for all 3 disks: Unable to identify the image format.
-
-Official notes:
-mess swtpc09 -flop1 DMAF2_BOOT.dsk
-press D to boot flex operating system. dir 0 will perform a directory of drive 0.
+** Notes on the use with common disk images **
+
+MAME emulates the floppy disk controller more thoroughly than some other
+common emulators for these systems. Other common emulators can ignore the disk
+density and disk side driven by the software. FLEX numbers sectors
+contiguously from one side to the other so a simple emulator need only map the
+track and sector numbers. Real disk images often mix single and double density
+formats, e.g. single density for the first track, and the software may not
+have been updated to correctly drive the side and density outputs, so disks
+for these other emulators might not not work in this MAME emulator. These disk
+images and their softare need to be updated - this is not working around
+limitations of this MAME emulator, these would not be expected to work on real
```

Comment:
“MAME” is not an adjective, and shouldn’t be genericised.  Please don’t use it that way as in “MAME emulator”.  MAME is a proper noun.

Reply:
Yes, done.